utils/watchcat/files/watchcat.sh

```diff
@@ -137,12 +152,12 @@ watchcat_monitor_network() {
 		for host in $ping_hosts; do
 			if [ "$iface" != "" ]; then
 				ping_result="$(
-					ping $ping_family -I "$iface" -s "$ping_size" -c 1 "$host" &> /dev/null
+					ping $ping_family -I "$iface" -s "$ping_size" -c 1 -W $ping_timeout "$host" &> /dev/null
```

Comment:
I advise to pass the `-W` param dependent upon the variable existing to avoid any weird empty variables which might break the ping command, in case another update breaks these long command strings. Standard usage elsewhere. `${parameter:+[word]}`

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html

e.g. 
```sh
${ping_timeout:+ -W $ping_timeout}
```

A good idea to fix all params to the ping command so. e.g.

```sh
					ping $ping_family ${iface:+-I $iface} ${ping_size:+-s $ping_size} -c 1 ${ping_timeout:+-W $ping_timeout} "$host" &> /dev/null
```

Use this technique everywhere you pass variables.

Reply:
done